### DISCUSSION TITLE
Does PyODBC Connect to DB2?

### DISCUSSION THREAD

**Discussion by anybodysguest** [mkleehammer/pyodbc#1153] [Q&A] [answered]
I think I know the answer to this question based on the PyODBC documentation but I'm looking for confirmation.  If I am trying to connect to DB2 using PyODBC, do I need to install the IBM DB2 ODBC drivers?

**Comment by v-chojas** [comment, marked as answer]
PyODBC lets you use any ODBC driver from Python, so the answer is yes.

**Reply by gordthompson** [reply]
>  If I am trying to connect to DB2 using PyODBC, do I need to install the IBM DB2 ODBC drivers?

Yes.
